page/NotificationForm.jsx

```diff
@@ -1,118 +1,209 @@
+	const notificationListener = useRef();
+	const responseListener = useRef();
+	const [text, onChangeText] = React.useState('');
+	const [number, onChangeNumber] = React.useState('');
 
-    useEffect(() => {
-        registerForPushNotificationsAsync().then(token =>
-            setExpoPushToken(token),
-        )
+	useEffect(() => {
+		registerForPushNotificationsAsync().then(token => setExpoPushToken(token));
 
-        notificationListener.current =
-            Notifications.addNotificationReceivedListener(notification => {
-                setNotification(notification)
-            })
+		notificationListener.current =
+			Notifications.addNotificationReceivedListener(notification => {
+				setNotification(notification);
+			});
 
-        responseListener.current =
-            Notifications.addNotificationResponseReceivedListener(response => {
-                console.log(response)
-            })
+		responseListener.current =
+			Notifications.addNotificationResponseReceivedListener(response => {
+				console.log(response);
+			});
 
-        return () => {
-            Notifications.removeNotificationSubscription(
-                notificationListener.current,
-            )
-            Notifications.removeNotificationSubscription(
-                responseListener.current,
-            )
-        }
-    }, [])
+		return () => {
+			Notifications.removeNotificationSubscription(
+				notificationListener.current,
+			);
+			Notifications.removeNotificationSubscription(responseListener.current);
+		};
+	}, []);
 
-    return (
-        <View
-            style={{
-                flex: 1,
-                alignItems: 'center',
-                justifyContent: 'space-around',
-            }}
-        >
-            <Text>Your expo push token: {expoPushToken}</Text>
-            <View style={{ alignItems: 'center', justifyContent: 'center' }}>
-                <Text>
-                    Title: {notification && notification.request.content.title}{' '}
-                </Text>
-                <Text>
-                    Body: {notification && notification.request.content.body}
-                </Text>
+	return (
+		<KeyboardAwareScrollView>
+			<SafeAreaView style={styles.container}>
+				<TextInput
+					style={styles.input1}
+					onChangeText={onChangeText}
+					value={text}
+					placeholder="Titulo"
+					keyboardType="text"
+				/>
+				<TextInput
+					style={styles.input}
+					onChangeText={onChangeNumber}
+					value={number}
+					placeholder="Descripción"
+					keyboardType="text"
+					multiline={true}
+					numberOfLines={10}
+				/>
+				<View
+					style={{
+						flex: 1,
+						alignItems: 'center',
+						justifyContent: 'space-around',
+					}}
+				>
+					<TouchableOpacity
+						style={styles.btnSend}
+						title="Enviar Notificación"
+						color="white"
+						onPress={async () => {
+							try {
+								await axios.post(
+									process.env.API_BASE_URL + '/api/v1/notification/',
+									{
```

Comment:
Why aren't you import this url from the `requests.js` file like the others???
I think this shall be in that file too.